### PR TITLE
Terra clinical error view

### PR DESCRIPTION
**Pull request by dkasper-was-taken**
### Summary
Terra Clinical Error View implementation.  This is a template for error messaging to the user.

### CODE REVIEW

File: packages/terra-clinical-error-view/lib/ErrorView.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  width: 100%;
+}
+
+.terraClinical-ErrorView-text {
+  @include terra-text-align-center();
+  color: $terra-grey-25;
+  flex: 0 0 auto;
+  font-size: 20px;
+  // font-size: 1.4285714285714286rem;
+  line-height: 30px;
```
Comment (emilyrohrbough) [inline packages/terra-clinical-error-view/lib/ErrorView.scss]: Stephen made a comment on my LabelValueView saying we are going to unit-less line-heights.

File: packages/terra-clinical-error-view/package.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-clinical/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-clinical#readme",
+  "devDependencies": {
+    "terra-toolkit": "^0.x"
+  },
+  "peerDependencies": {
```
Comment (emilyrohrbough) [inline packages/terra-clinical-error-view/package.json]: How should we be setting up the peerDependencies vs dependencies in these packages we are creating? These differ from your NoDataView pull request and I am pretty certain they differ from how I set them up in LabelValueView.

Comment (dkasper-was-taken) [inline packages/terra-clinical-error-view/package.json]: Nice catch, I was missing the peer dependencies.

File: packages/terra-clinical-error-view/lib/ErrorView.scss [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  padding-top: 4px;
+}
+
+.terraClinical-ErrorView-glyph {
+  flex: 0 0 auto;
+}
+
+.terraClinical-ErrorView-button {
+  flex: 0 0 auto;
+  padding-top: 4px;
```
Comment (StephenEsser) [inline packages/terra-clinical-error-view/lib/ErrorView.scss]: Should we keep units consistent? I think rem is the only unit being used in terra-core

File: packages/terra-clinical-error-view/docs/README.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+```jsx
+import React from 'react';
+import ErrorView from 'terra-clinical-error-view';
+
+<ErrorView
+  name="test name"
+  description="test description"
+  button={<Button />}
+  isGlyphHidden={false}
+ />
```
Comment (StephenEsser) [inline packages/terra-clinical-error-view/docs/README.md]: This example seems out of date, it doesn't look like the ErrorView accepts a button prop anymore

File: packages/terra-clinical-error-view/src/ErrorView.scss [hunk 5]
```diff
@@ -8,17 +8,17 @@
...
+  padding-bottom: 0.2857rem;
+}
+
 .terraClinical-ErrorView-text {
   @include terra-text-align-center();
   color: $terra-grey-25;
   flex: 0 0 auto;
   font-size: 1.7142857142857142rem;
   line-height: 1.284;
-  padding-top: 0.2857rem;
```
Comment (emilyrohrbough) [inline packages/terra-clinical-error-view/src/ErrorView.scss]: Did you still want padding-top?

Comment (dkasper-was-taken) [inline packages/terra-clinical-error-view/src/ErrorView.scss]: It was moved to a padding bottom on the glyph element, as the glyph is optional.

Comment (emilyrohrbough) [inline packages/terra-clinical-error-view/src/ErrorView.scss]: Ok. Didn't want it to accidentally get dropped :)

**Pull request merged**